regolith/helpers/u_milestonehelper.py

```diff
@@ -81,12 +81,11 @@ def db_updater(self):
             item['identifier'] = 'milestones'
         all_milestones = [deliverable, kickoff]
         all_milestones.extend(milestones)
-        all_milestones.sort(key=lambda x: x['due_date'], reverse=False)
-        index = 2
-        numbered_milestones = {}
-        for item in all_milestones:
-            numbered_milestones[str(index)] = item
-            index += 1
+        for i in all_milestones:
+            i['due_date'] = get_due_date(i)
+        all_milestones.sort(key = lambda x: x['due_date'], reverse=False)
+        index = list(range(2, (len(all_milestones)+2)))
+        numbered_milestones = dict(zip(index, all_milestones))
```

Comment:
I don't think you need to make a dict here.

I think a number of updaters are taking the same approach so maybe we want some functions in tools.py that do all this work.....  We will discuss in group meeting today

Reply:
ok! I deleted the dic!
it is passing tests locally, but in the CI it is getting annoyed with the dates ''. 
do you have any idea of why there is a different behavior?